lib/validateCreateArgs.js

```diff
@@ -177,19 +178,21 @@ function validateCreateArgs(args) {
         if (args.meta && args.meta['wt-compiler'] && args.meta['wt-compiler'] !== MIDDLWARE_COMPILER) {
             throw Cli.error.invalid('Use of middleware is incompatible with a custom webtask compiler.');
         }
-
+        const filter = mw => mw.startsWith('http') ? 'urls' : 'modules';
+        const { urls, modules } = _.groupBy(args.middleware, filter);
```

Comment:
Again, we can't rely on destructuring object assignment across node versions that we support.